src/main/scala/uk/gov/nationalarchives/dp/client/EntityClient.scala

```diff
@@ -140,7 +140,7 @@ object EntityClient {
 
     private def createUpdateRequestBody(
         ref: Option[UUID],
-        titleToChange: Option[String],
+        titlePotentiallyChange: String,
```

Comment:
I guess this is called "potentiallyChange" because it's possible that it's the same title but this is always going to be passed to the update request so would `title` not be better?